doc/releasenotes/v0.5.0.rst

```diff
@@ -115,13 +115,13 @@ Python API Changes
     - ``template_inputfile_path`` → ``template_input_file_path``
     - Changed `iter_inputfile` and `iter_matfile` to be attributes instead of parameters
 
-  - ``DepcodeSerpent``
+  - ``DepcodeSerpent`` → ``SerpentDepcode``
 
     - ``template_inputfile_path`` → ``template_input_file_path``
     - Changed `iter_inputfile` and `iter_matfile` to be attributes instead of parameters
 
 
-  - ``DepcodeOpenmc`` is a ``Depcode`` subclass that interfaces with ``openmc``. This class implements the following functions 
+  - ``OpenMCDepcode`` is a ``Depcode`` subclass that interfaces with ``openmc``. This class implements the following functions 
```

Comment:
Is there any reason you have one with the arrow and one without? Is `OpenMCDepcode` new? I thought you were renaming it.

Reply:
`OpenMCDepcode` doesn't exist in v0.4.0 yet, so there's no need to indicate that the name has been changed inside the development period between v0.4.0 and v0.5.0.

Reply:
Ok. That makes sense.